app/src/main/assets/html/error/css/styles.css

```diff
@@ -0,0 +1,173 @@
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+/* open-sans-regular - latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Open Sans Regular'), local('OpenSans-Regular'), url('../fonts/open-sans/open-sans-v15-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */ url('../fonts/open-sans/open-sans-v15-latin-regular.woff') format('woff');
+  /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+/* open-sans-700 - latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Open Sans Bold'), local('OpenSans-Bold'), url('../fonts/open-sans/open-sans-v15-latin-700.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */ url('../fonts/open-sans/open-sans-v15-latin-700.woff') format('woff');
+  /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+* {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 24px;
+  line-height: 1.2em;
+}
+html,
+body {
+  display: grid;
+  grid-template-rows: auto;
+  justify-items: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  background: #f4f4f4;
+}
+a {
+  color: black;
+  text-decoration: none;
+}
+a:hover {
+  text-decoration: underline;
+  text-decoration-style: dotted;
+}
+hr {
+  margin: 30px 0;
+}
+h1 {
+  font-size: 60px;
+  font-weight: bold;
+  text-align: center;
+}
+h2 {
+  font-size: inherit;
+  font-weight: bold;
+}
+p {
+  margin: 1em 0em;
+}
+ol {
+  list-style-type: decimal;
+}
+ol,
+ul {
+  list-style-position: inside;
+}
+li {
+  margin: 1em 0;
+  text-indent: -1em;
+  padding-left: 1em;
+}
+button {
+  border: 2px solid #434c58;
+  border-radius: 10px;
+  background: #434c58;
+  cursor: pointer;
+  color: white;
+  font-weight: bold;
+  padding: 0 30px;
+  height: 80px;
+}
+button:hover {
+  color: #434c58;
+  background: #e3e4e5;
+}
+button:active {
+  background: black;
+  color: white;
+  border-color: black;
+}
+[data-type="danger"] {
+  background: url('../images/icon-errorpage-thisconnectionisuntrusted.svg');
```

Comment:
these three lines can be condensed to this one:

```
  background: url(../images/icon-errorpage-thisconnectionisuntrusted.svg) repeat-x;
```

Reply:
out of curiosity, why is this the only one that uses CSS-referenced `background-image`s, as opposed to all the others using `<img>`s?

Reply:
I used background image to keep margins and centering unaffected for consistency across the pages for layout.

Reply:
gotcha, feel free to use whatever works - I was just surprised/confused by it at first